isvctl/configs/suites/README.md

```
@@ -265,7 +265,7 @@ its plan item is not platform-scoped.
 | `query_repair_history` | test | `providers/nico/scripts/breakfix/query_repair_history.py` | `history_queryable`, `records[].{machine_id,entries}` -- a record needs non-empty `entries` to count (BFX02-03) |
 | `query_switch_firmware` | test | `providers/my-isv/scripts/breakfix/query_switch_firmware.py` | `trays[].{tray_id,firmware_version}` (BFX03-02) |
 | `query_bmc_kernel_logs` | test | `providers/nico/scripts/breakfix/query_bmc_kernel_logs.py` | `hosts[].{host_id,kernel_log_available}` (BFX03-03) |
-| `return_node_maintenance` | test | `providers/my-isv/scripts/breakfix/return_node_maintenance.py` | `operation.{requested,accepted,machine_id,maintenance_mode}` (BFX01-02) |
+| `return_node_maintenance` | test | `providers/my-isv/scripts/breakfix/return_node_maintenance.py` template; `providers/shared/breakfix/return_node_maintenance.py` Kubernetes Maintenance Operator reference | `operation.{requested,accepted,machine_id|node_id,maintenance_mode,restored}`; Kubernetes also requires `workload_evacuated`, `replacement_blocked`, and `workload_recovered` (BFX01-02) |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)